container_service_extension/cse.py

```diff
@@ -25,46 +25,58 @@
 def cli(ctx):
     """Container Service Extension for VMware vCloud Director.
 
-\b
-    Manages CSE.
 \b
     Examples
-        cse sample
-            Generate sample config.
+        cse version
+            Display CSE version.
 \b
         cse sample > config.yaml
-            Save sample config.
+            Generate sample CSE config in a file named 'config.yaml'.
 \b
-        cse check
-            Validate CSE installation/configuration.
+        cse install
+            Install CSE using data from 'config.yaml'.
 \b
-        cse install --config config.yaml
-            Install CSE.
+        cse install -c myconfig.yaml --template photon-v2
+            Install CSE using data from 'myconfig.yaml' but only create
+            template 'photon-v2'.
 \b
-        cse install --config config.yaml --template photon-v2
-            Install CSE. It only creates the template specified.
+        cse install --update
+            Install CSE, and if the templates already exist in vCD, create
+            them again.
 \b
-        cse install --config config.yaml --no-capture --ssh-key  \\
-                    ~/.etc/id_rsa.pub
-            Install CSE. The temporary vApp specified in the config file \\
-            will be created, but the vApp will not be captured as a template \\
-            in the catalog. --ssh-key option is required if --no-capture \\
-            is used
+        cse install --no-capture --ssh-key ~/.ssh/id_rsa.pub
+            Install CSE, but don't capture temporary vApps to a template.
+            Instead, leave them running for debugging purposes. Copy specified
+            SSH key into all VMs so users with the cooresponding private key
+            have access (--ssh-key is required when --no-capture is used).
 \b
-        cse install --config config.yaml --template photon-v2 --update \\
-                    --amqp skip --ext skip
-            Update the specified template.
+        cse install --amqp skip --ext config
+            Install CSE, but skip amqp configuration step and register CSE to
+            vCD without prompting.
 \b
-        cse version
-            Display version.
+        cse check
+            Validate CSE was installed correctly according to 'config.yaml'.
```

Comment:
cse check -  simply checks whether info in config.yaml is accurate or not. It does not guarantee CSE installation.

You can mention in detail what actually it does to avoid confusion.
- Ensures if vCD, VC, AMQP are reachable.
- Ensures if a given template is already created?